base/prefs.c

```diff
@@ -174,7 +174,8 @@ prefs_config (const char *config)
   if (!global_prefs)
     prefs_init ();
 
-  strncpy (buffer, config, sizeof (buffer));
+  memset (buffer, 0, sizeof (buffer));
+  memcpy (buffer, config, sizeof (buffer));
```

Comment:
```suggestion
  memset (buffer, 0, sizeof (buffer));
  memcpy (buffer, config, sizeof (buffer) - 1);
```

we might want to have a null termination at the end?

I have seen in the other places the usage of `bzero` ... we could use it here too?

Reply:
I would rather use the standard memset tbh